spec/features/idv/analytics_spec.rb

```diff
@@ -8,7 +8,7 @@
   let(:fake_analytics) { FakeAnalytics.new }
```

Comment:
IIRC this spec was here to maintain continuity between analytics events with FSMv2 being put together. Is there any value here with FSMv2 gone or could we remove this entire spec file?

Reply:
Personally, I still see value in asserting the events that we're expecting to have been logged during identity verification. I also like how this is implemented as a single canonical list, though I wouldn't mind if they were part of feature specs for the individual steps if we could trust that we did so exhaustively (I don't trust this, FWIW). 

I had actually planned to expand this spec to include events logged during IAL1 authentication, as a regression spec for [an issue I recently identified](https://github.com/18F/identity-idp/pull/6946#discussion_r976534843).

Reply:
+1 this spec is also useful if we wanted to refactor/change out FSM but keep the analytics events we have for consistency